examples/pipelines/snyk-project/pipelinerun.yaml

```diff
@@ -7,10 +7,18 @@ spec:
   pipelineRef:
     name: dracon-snyk-project
   params:
+  - name: git-clone-url
+    value: https://github.com/adeyosemanputra/pygoat.git
   - name: producer-snyk-docker-api-key
-    value: ""
+    value: "$snyk-api-key"
```

Comment:
this value looks a bit sus. is it supposed to be just an example?

Reply:
it is very much an example and on purpose being a $variable